docs/source/blocks/index.md

```diff
@@ -15,6 +15,7 @@ introduction
 anatomy
 settings
 editcomponent
+blockStyleWrapper
```

Comment:
Let's use lower-case only. Also consider using dashes for SEO. Also rename the file to which this refers.

```suggestion
block-style-wrapper
```